packages/camera/camera_avfoundation/ios/Classes/FLTCam.m

```diff
@@ -1101,8 +1126,23 @@ - (BOOL)setupWriterForPath:(NSString *)path {
     return NO;
   }
 
-  NSDictionary *videoSettings = [_captureVideoOutput
-      recommendedVideoSettingsForAssetWriterWithOutputFileType:AVFileTypeMPEG4];
+  NSMutableDictionary *videoSettings = [[_captureVideoOutput
+      recommendedVideoSettingsForAssetWriterWithOutputFileType:AVFileTypeMPEG4] mutableCopy];
+
+  if (_videoBitrate || _fps) {
+    NSMutableDictionary *compressionProperties = [@{} mutableCopy];
```

Comment:
this can be just `[NSMutableDictionary dictionary]`

Reply:
Replaced with `[[NSMutableDictionary alloc] init]`. Ok?

Reply:
Both are fine.